src/sdk/k4a.c

```diff
@@ -1018,15 +1019,58 @@ k4a_result_t k4a_transformation_depth_image_to_color_camera(k4a_transformation_t
     k4a_transformation_image_descriptor_t depth_image_descriptor = k4a_image_get_descriptor(depth_image);
     k4a_transformation_image_descriptor_t transformed_depth_image_descriptor = k4a_image_get_descriptor(
         transformed_depth_image);
+    k4a_transformation_image_descriptor_t dummy_descriptor = { 0 };
 
     uint8_t *depth_image_buffer = k4a_image_get_buffer(depth_image);
+    uint8_t *custom_image_buffer = NULL;
     uint8_t *transformed_depth_image_buffer = k4a_image_get_buffer(transformed_depth_image);
+    uint8_t *transformed_custom_image_buffer = NULL;
+
+    return TRACE_CALL(transformation_custom_depth_image_to_color_camera(transformation_handle,
+                                                                        depth_image_buffer,
+                                                                        &depth_image_descriptor,
+                                                                        custom_image_buffer,
+                                                                        &dummy_descriptor,
+                                                                        transformed_depth_image_buffer,
+                                                                        &transformed_depth_image_descriptor,
+                                                                        transformed_custom_image_buffer,
+                                                                        &dummy_descriptor,
+                                                                        K4A_TRANSFORMATION_INTERPOLATION_TYPE_LINEAR,
```

Comment:
may be add a comment to specify these two parameter in this function are noop?

Reply:
Yeah, I can add a comment.